plugins/catalog-backend/api-report.md

```diff
@@ -764,6 +765,10 @@ export class DefaultCatalogCollator implements DocumentCollator {
     },
   ): DefaultCatalogCollator;
   // (undocumented)
+  protected getDocumentText(entity: Entity): string;
+  // (undocumented)
+  protected isUserEntity(entity: Entity): entity is UserEntity;
+  // (undocumented)
```

Comment:
A little torn on whether these should be marked `protected` here vs. `private` (and thus not part of the API).  On the one hand, it could be nice to have a little hook to modify the text field logic via inheritance.  ...But on the other hand, I don't know if we want to encourage that pattern of customization (vs. writing your own collator).  Whaddya think?

Reply:
Thank you for catching that! I'd lean towards making them `private` to avoid that. Specially `isUserEntity`, don't think anyone needs to modify that...
What about the other methods in this class? Most of them are `protected`.

Reply:
😬 &nbsp;yeah probably most of the methods/properties should have been `private` but we just weren't thinking about it / vigilant.  

We can probably find ways to deprecate and eventually remove them in another PR?  But for now, I'd agree with making these new ones private out the gate.